package.json

```diff
@@ -50,86 +50,87 @@
     "lint": "npm run standard",
     "lint:fix": "npm run standard:fix",
     "audit:check": "npx audit-ci --config ./audit-ci.jsonc",
+    "audit:fix": "npm audit fix",
     "dep:check": "npx ncu -e 2",
     "dep:update": "npx ncu -u",
     "release": "npx standard-version --no-verify --releaseCommitMessageFormat 'chore(release): {{currentTag}} [skip ci]'",
     "snapshot": "npx standard-version --no-verify --skip.changelog --prerelease snapshot --releaseCommitMessageFormat 'chore(snapshot): {{currentTag}}'",
     "genexec": "pkg -t node8-win ."
   },
   "dependencies": {
-    "@elastic/elasticsearch": "^8.6.0",
-    "@hapi/basic": "7.0.1",
+    "@elastic/elasticsearch": "^8.9.0",
+    "@hapi/basic": "7.0.2",
     "@hapi/boom": "10.0.1",
     "@hapi/good": "9.0.1",
-    "@hapi/hapi": "^21.3.1",
+    "@hapi/hapi": "^21.3.2",
     "@hapi/inert": "7.1.0",
-    "@hapi/vision": "7.0.1",
-    "@mojaloop/central-services-logger": "^11.2.0",
-    "@mojaloop/ml-testing-toolkit-shared-lib": "^13.0.0",
-    "@mojaloop/sdk-standard-components": "17.1.1",
+    "@hapi/vision": "7.0.3",
+    "@mojaloop/central-services-logger": "^11.2.2",
+    "@mojaloop/ml-testing-toolkit-shared-lib": "^14.0.0",
+    "@mojaloop/sdk-standard-components": "17.1.3",
     "@now-ims/hapi-now-auth": "2.1.0",
     "@types/socket.io": "^3.0.1",
     "adm-zip": "^0.5.10",
     "atob": "2.1.2",
-    "axios": "^0.27.2",
-    "chai": "^4.3.7",
+    "axios": "^1.5.0",
+    "chai": "^4.3.8",
```

Comment:
Can we get rid of carrot symbol (^) for all the dependencies as we are following for other repositories?